basic_functions.cpp

```diff
@@ -21,6 +21,16 @@ int mul(int a, int b)
     return a * b;
 }
 
+
+//Function to divide two numbers.
+//Here we divide two numbers.
+// Another function in our program.
```

Comment:
```suggestion
//Function to divide two numbers.
```

I think all other lines are not necessary.

Reply:
Accepting the suggestion.